packages/gitbook/src/lib/api.ts

```diff
@@ -126,18 +125,25 @@ export function withAPI<T>(client: GitBookAPI, fn: () => Promise<T>): Promise<T>
     return apiSyncStorage.run(client, fn);
 }
 
+// FIXME: Fix @gitbook/api types for PublishedSiteContentLookup
```

Comment:
Still true?

Reply:
Yes, it'll be a small change in our API type to be able to extract this type for `SpaceContentLookup` because currently it's an union of `{  redirect: '...' } | { site: '...' }`, I want the latter one to be it's own type